src/vue/modules/movements-history/components/effect-viewers/balance-changed.vue

```diff
@@ -8,28 +8,47 @@
         {{ amount | formatMoney }}
       </td>
     </tr>
-    <tr class="attributes-viewer__table-row">
+    <tr
+      v-if="fixedFee.value > 0"
+      class="attributes-viewer__table-row"
+    >
       <td class="attributes-viewer__table-cell">
         {{ 'movements-history.fixed-fee-lbl' | globalize }}
       </td>
       <td class="attributes-viewer__table-cell">
         {{ fixedFee | formatMoney }}
       </td>
     </tr>
-    <tr class="attributes-viewer__table-row">
+    <tr
+      v-if="calculatedPercentFee.value > 0"
+      class="attributes-viewer__table-row"
+    >
       <td class="attributes-viewer__table-cell">
         {{ 'movements-history.calculated-percent-fee-lbl' | globalize }}
       </td>
       <td class="attributes-viewer__table-cell">
         {{ calculatedPercentFee | formatMoney }}
       </td>
     </tr>
+    <tr
+      v-if="totalFee.value > 0"
+      class="attributes-viewer__table-row"
+    >
+      <td class="attributes-viewer__table-cell">
+        {{ 'movements-history.total-fee-lbl' | globalize }}
+      </td>
+      <td class="attributes-viewer__table-cell">
+        {{ totalFee | formatMoney }}
+      </td>
+    </tr>
```

Comment:
all fee related rows should be shown if totalFee > 0, do not hide them one-by-one. so we provide the user detailed information of what kind of fees were charged, what were not.

Reply:
Updated